backend/models/league.go

```diff
@@ -2,6 +2,7 @@ package models
 
 //League will be for the league that will be using the ladder as a basic starting off point.
 type League struct {
+	ID     int    `json:"ID"`
```

Comment:
Shouldn't this be lower case? Does it map to the `mockleague.json` data?

Reply:
fixed